package.json

```diff
@@ -11,6 +10,7 @@
     "release:dry-run": "semantic-release --dry-run",
     "start": "nodemon -r dotenv/config scripts/start.js",
     "serve": "parcel scripts/index.html",
+    "serve:start": "parcel scripts/start.html",
```

Comment:
Not sure we want to add this as part of these PR. Additionally, I don't think we should have 2 "samples" running. Let's discuss this separately